index.html

```diff
@@ -203,7 +246,8 @@ <h2 class="contact-comment">I'm always interested in hearing about new projects,
         </li>
         <li>
           <label for="msg"></label>
-          <textarea id="msg" name="user_message" required maxlength = "500" placeholder="Enter text here"></textarea>  
+
+          <textarea id="msg" name="user_message" required maxlength = "500" placeholder="Hello Sunga. I would like to get in touch with you..."></textarea>  
```

Comment:
The underlines in the form inputs are missing. You can add them using `border-bottom`

This is your design:
![image](https://user-images.githubusercontent.com/61361037/213743409-04d5b173-d24d-427c-ba67-985ce6ab422a.png)
This is FIGMA design:
![image](https://user-images.githubusercontent.com/61361037/213743285-4d12d3ee-1771-477a-9cb7-1404ad9c68ce.png)

Reply:
hi @RaoAkif 

I did try to use border-bottom , but each time i did, it brought, the entire border around the text, so I wasnt sure what else to do.